player/public/coreplugins/org.sunbird.player.endpage-1.1/renderer/plugin.js

```diff
@@ -23,12 +23,11 @@ org.ekstep.contentrenderer.baseEndepage.extend({
     },
     gotTohome: function() {
         if (!isbrowserpreview) {
-            EkstepRendererAPI.hideEndPage();
             stageId = !_.isUndefined(Renderer) ? Renderer.theme._currentStage : " ";
             TelemetryService.interact("TOUCH", "gc_genie", "TOUCH", {
                 stageId: stageId
             });
-            exitApp();
+            genieservice.endGenieCanvas();
```

Comment:
What happen to the logic inside exitApp(). 
endGenieCanvas() method should be there in every service(web, local & mobile services)

Reply:
The exitApp() method is for mobile only, Previously it was closing the Cordova activity, but now we are showing exit popup from mobile(which will close the player window), I need to add this method here.